certified-connectors/DataMotion/README.md

```diff
@@ -1,5 +1,5 @@
-# Secure Message Delivery Connector
-DataMotion, Inc. specializes in secure data exchange APIs, providing companies the ability to integrate secure message and file exchange solutions into their existing workflows and automated processes. DataMotion’s secure message delivery connector enables seamless, one-way encrypted messaging, helping users protect their critical data and meet compliance regulations.
+# DataMotion Connector
```

Comment:
@plmvas It is not allowed to change the connector name when your connector has any usage. If let us know if your connector 0 usages, and please do not change the connector folder name as it is not allowed.

Reply:
Hi Sriyen,

In the Connector Telemetry, we do not see any uses of the connector. We were also told by a representative in the ISV Studio that a rename of the connector is okay due to it being pretty new. 

As for the name of the connector folder, we haven't changed it.

Regards,
Chris

Reply:
@plmvas Thank you for the clarification, You can change the title of your connector, but can you please change the folder name to "Secure Message Delivery" as it needs to be matched with your connector submission (API) name.

Reply:
Hi Sriyen,

No problem! Where can I find the connector submission name? In GitHub, the folder name is "DataMotion", so it would match the new connector name. If the connector name matches the folder name, do we still need to change it?

Regards,
Chris

Reply:
@plmvas Please change the folder name of your connector in the GitHub repo.

Reply:
@sriyen-msft For clarification, the folder name of the connector will need to be "Secure Message Delivery" while the actual name of the connector is "DataMotion"? They both can't be "DataMotion"?

Reply:
@plmvas As per the design, the connector folder name should be same as the connector title or API name. "Datamotion" is nowhere related to the connector title or API name. Hence it is hard to find your connector in the repository.

Reply:
@sriyen-msft In this case, we're changing the connector title to be "DataMotion". Under "certified-connectors", the folder name is already "DataMotion". Does this not meet the design specifications?

Reply:
@sriyen-msft Hi Sriyen, I wanted to follow up on this. How would you like to proceed?